angular-workspace/projects/ni/nimble-angular/src/directives/table-column/text/nimble-table-column-text.directive.ts

```diff
@@ -30,6 +30,26 @@ export class NimbleTableColumnTextDirective {
         this.renderer.setProperty(this.elementRef.nativeElement, 'fieldName', value);
     }
 
+    public get actionMenuSlot(): string | undefined {
```

Comment:
Much like we had to do with Blazor when we created the `TableColumn` component to have `TableColumnText` extend, I think we should have a base `NimbleTableColumnDirective` that all of this implementation (and `columnId`) should move to (sorry I missed that on the PR that introduced `columnId` into this directive).